rabbitmq/assets/configuration/spec.yaml

```diff
@@ -9,7 +9,6 @@ files:
   - template: instances
     options:
     - name: rabbitmq_api_url
-      required: true
```

Comment:
I have a todo to add an explicit test that this is present when using the management/legacy check. Based on some exploratory testing I think the validation is already present, but would like to confirm.

Reply:
Will the openmetrics implementation use this same config option as the "openmetrics_endpoint"? If so, we should keep `required: true`.